Demo/MozillaSocial-iOS/MoSoContent/LoginView.swift

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
```

Comment:
can we remove this `import`? `Foundation` becomes redundant if you `import SwiftUI`